### ISSUE TITLE
[gulliver] Improve documentation (Trac #1339)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1331]
Documentation for seed service is missing.

Migrated from https://code.icecube.wisc.edu/ticket/1339
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "Documentation for seed service is missing.", 
    "reporter": "kkrings", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[gulliver] Improve documentation", 
    "priority": "major", 
    "keywords": "", 
    "time": "2015-09-11T07:21:37", 
    "milestone": "", 
    "owner": "kkrings", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150911T07:22:09: kkrings changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20150918T21:14:47: 
* kkrings commented:

> In changeset 137756:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="137756"
> Added some documentation for seed services; closes #1331.
> ```


* kkrings changed status from "accepted" to "closed"
* kkrings changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas commented:

> Milestone IceRec V5 deleted


* olivas changed milestone from "IceRec V5" to ""
